tools/tf2openapi/types/tfsavedmodel.go

```diff
@@ -17,11 +18,23 @@ type TFSavedModel struct {
 }
 
 func NewTFSavedModel(model pb.SavedModel) TFSavedModel {
-	return TFSavedModel{}
+	if tfMetaGraphs := extractMetaGraphs(model.MetaGraphs); len(tfMetaGraphs) == 0 {
+		panic("No graph to serve from SavedModel.")
+	} else {
+		return TFSavedModel{
+			MetaGraphs: tfMetaGraphs,
+		}
+	}
 }
 
 func extractMetaGraphs(metaGraphs []*pb.MetaGraphDef) []TFMetaGraph {
```

Comment:
This logic is so clean that it may be cleaner to just do this inline 

```
tfMetaGraphs := []TFMetaGraph{}
for _, metaGraph := range metaGraphs {
	if utils.Includes(metaGraph.MetaInfoDef.Tags, ServingMetaGraphTag) {
		tfMetaGraphs = append(tfMetaGraphs, NewTFMetaGraph(*metaGraph))
	}
}
if len(tfMetaGraphs) == 0 {
		panic("No graph to serve from SavedModel.")
}
return TFSavedModel{
	MetaGraphs: tfMetaGraphs,
}
```